sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1326,14 +1326,24 @@ object SQLConf {
 
   val ARROW_EXECUTION_ENABLED =
     buildConf("spark.sql.execution.arrow.enabled")
-      .doc("When true, make use of Apache Arrow for columnar data transfers." +
-        "In case of PySpark, " +
+      .doc("(Deprecated since Spark 3.0, please set 'spark.sql.pyspark.execution.arrow.enabled'.)")
+      .booleanConf
+      .createWithDefault(false)
+
+  val PYSPARK_ARROW_EXECUTION_ENABLED =
+    buildConf("spark.sql.pyspark.execution.arrow.enabled")
```

Comment:
spark.pyspark.execution.arrow.enabled ?

Reply:
Actually that s what I tired first but here is in SQLConf.scala. if we go for Pyspark or Sparkr prefix, those configurations should be SparkConf under Python.scala, for instance.